web/src/components/shared/Ads/Ad.tsx

```diff
@@ -29,33 +29,84 @@ import { useFetchTraderInfo } from 'web/src/hooks/data/useFetchTraderInfo';
 import { useTrustUser } from 'web/src/hooks/mutations/useTrustUser';
 import { useTradeEstimate } from 'web/src/hooks/data/useTradeEstimate';
 import { useDebouncedCallback } from 'use-debounce';
+import ArrowDropDown from 'web/src/assets/svg/ArrowDropDown.svg';
+import cn from 'classnames';
+import * as s from 'web/src/components/shared/Modal/TradeDetailsModal.css';
+import * as adStyle from 'web/src/components/shared/Ads/Ad.css';
```

Comment:
```suggestion
import * as adStyle from './Ad.css';
```
Файл соседний, не нужно использовать абсолютный путь

Reply:
done